CHANGELOG.md

```diff
@@ -0,0 +1,22 @@
+All notable changes to this project will be documented in this file.
+This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
+
+# [v1.0.0] (Unreleased)
+### Notes
+  Major release which extends support to OneView appliances with versions 4.10, using the OneView Rest API version 800.
+
+### Major changes:
+- Extended support of SDK to OneView API800.
+- Support for Go 1.11
+- Support for Terraform 0.11    
```

Comment:
we have not updated the terraform yet..though it should be our plan in future. So, remove this as of now.